### PR TITLE
CI-2066 doc update introducing AEM 6.4 support

### PR DESCRIPTION
**Pull request by francoisledroff**

### CODE REVIEW

File: events/using/aem-event-setup.md [hunk 1]
```diff
@@ -26,14 +26,19 @@ Before setting up and using AEM with Adobe I/O, you will need to do the followin
 
 To complete this solution, you will need authorization to use the following services:
 
-*   An AEM instance, version 6.2 or 6.3, with administrative permissions. (**Note:** AEM screens in this topic are captured from version 6.3.)
+*   An AEM instance, version `6.2.x`, `6.3.x` or `6.4.x`, with administrative permissions. (**Note:** AEM screens in this topic are captured from version 6.3.)
 *   [Adobe I/O Console](https://adobe.io/console) access, with administrative permissions for your enterprise organization. If you do not have administrative permissions, please contact [ioevents-beta@adobe.com](mailto:ioevents-beta@adobe.com). After requesting administrative permissions, watch for an email from Adobe Systems Incorporated, notifying you of your permissions:
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: I'm pretty sure we want to change this. We are no longer in beta, *and* we are not the ones in charge of giving admin access to Console. @cxs504 how do you suggest we rephrase this section?

File: events/using/aem-event-setup.md [hunk 2]
```diff
@@ -26,14 +26,19 @@ Before setting up and using AEM with Adobe I/O, you will need to do the followin
...
 
       ![Admin rights email](../../img/events_aem_01.png "Admin rights email")
 
 ### Register an AEM event consumer app
 
-You will need to register an AEM event consumer app, such as a webhook, to see responses to AEM changes. These instructions include steps for setting up a webhook that is able to accept and reply to a [challenge HTTP request](../intro/webhook_docs_intro.md##thechallengerequest) parameter sent by Adobe I/O Channel & Subscription Management (CSM). For more information on understanding and working with webhooks, see the [Introduction to Adobe I/O Events Webhooks](../intro/webhook_docs_intro.md).
+You will need to register an AEM event consumer app, such as a webhook, to see responses to AEM changes. 
+These instructions include steps for setting up a webhook that is able to accept 
+and reply to a [challenge HTTP request](../intro/webhook_docs_intro.md##thechallengerequest) parameter sent 
+by Adobe I/O Channel & Subscription Management (CSM).
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: Please, let's change this, as CSM is an internal implementation detail. Better: "... sent by I/O Events"

File: events/using/aem-event-setup.md [hunk 3]
```diff
@@ -203,7 +211,42 @@ To create an [Adobe I/O Console](https://adobe.io/console) integration:
...
+   ![Adobe IMS Configurations UI](../../img/events_aem_adobe-ims-conf-1.png "Adobe IMS Configurations UI")
+   
+2. Click **Create**; 
+   **Select `AdobeIOEvents`** in the Cloud Solution drop down; 
+   once selected, you should see the certificate you just added to AEM `eventproxy-service` user's keystore;
+   Click **Next** 
+   
+    ![Adobe IMS Configuration Creation](../../img/events_aem_adobe-ims-conf-2.png "Adobe IMS Configuration Creation")
+
+3. Fill in the various entry expected to configure the IMS account associated with the integration
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: entries

File: events/using/aem-event-setup.md [hunk 4]
```diff
@@ -203,7 +211,42 @@ To create an [Adobe I/O Console](https://adobe.io/console) integration:
...
+   once selected, you should see the certificate you just added to AEM `eventproxy-service` user's keystore;
+   Click **Next** 
+   
+    ![Adobe IMS Configuration Creation](../../img/events_aem_adobe-ims-conf-2.png "Adobe IMS Configuration Creation")
+
+3. Fill in the various entry expected to configure the IMS account associated with the integration 
+ you just created in the Adobe I/O Console
+
+*   For **Title**: specify **Adobe IO Events** (or any other title that makes sense to you).
+*   For **Authorization Server**: it should be `https://ims-na1.adobelogin.com` (unless the url shown in the **JWT** tabulation of your integration page in the Adobe I/O Console is different),
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: period at the end, not comma

Comment (rperelma) [reply events/using/aem-event-setup.md]: tab, not tabulation (same below)

File: events/using/aem-event-setup.md [hunk 5]
```diff
@@ -228,25 +271,26 @@ To configure Adobe I/O Events as a cloud service in AEM:
 *   For **Organization ID**: Provide the ID shown on the Adobe I/O Console.
 *   For **Client Secret**: AEM will automatically retrieve the value from the Adobe I/O Console.
 
+
 ### Perform an AEM health and configuration check
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: This part remains relevant to 6.4, right? The current formatting, makes it see only applicable to 6.2 and 6.3

File: events/using/aem-event-setup.md [hunk 6]
```diff
@@ -316,28 +360,23 @@ Note: Once you have registered your webhook, responses will include a [status](h
...
+for page related events, publish or unpublish a page;
+finally to test our custom OSGI event type sample, you may use our custom OSGI event health check.
+    
+  ![Custom osgi event health check](../../img/events_aem_28.png "Custom osgi event health check")
 
-1. Check that events are sent to and received by Adobe I/O Event receiver (the AEM ingress adapter). To do this, execute the health check tagged with **`eventproxy`, `eventreceiver`**.
+ 
+If for some reason, your webhook is failing, note that the Adobe I/O console holds a **Debug Tracing** feature:
+It allows you to watch all the events payloads emitted by Adobe I/O towards your webhook and the associated webhook response.
+In the console, select your integration, the `Events` Tabulation, your webhook, then `Debug tracing`,
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: tab, not tabulation

File: events/using/aem-event-setup.md [hunk 7]
```diff
@@ -358,7 +397,9 @@ For each of these you can edit:
   
 The various OSGI event handlers will intercept the events according to these values and then map these OSGI events to the Adobe I/O Event Model before forwarding them to Adobe I/O.
 
-The solution leverages the OSGI configuration factory pattern, hence can you not only edit these configurations, but you can also remove and add such configurations.
+The solution leverages the OSGI configuration factory pattern, 
+hence can you not only edit these configurations, but you can also remove
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: hence, you can not only...  (instead of "can you")

File: events/using/aem-event-setup.md (hunk 5 above)
Comment (francoisledroff) [reply events/using/aem-event-setup.md]: yes it's applicable to all

File: events/using/aem-event-setup.md (hunk 1 above)
Comment (francoisledroff) [reply events/using/aem-event-setup.md]: I removed the beta paragraph...

**Comment by rperelma** [comment]
LGTM  👍

File: events/using/aem-event-setup.md [hunk 8]
```diff
@@ -411,6 +411,48 @@ To configure using the panel:
 
 3. For **Adobe I/O Events OSGI to XDM Event Mapping Configuration**, select **+**, **Edit**, or **Delete**.
 
+
+#### Adobe I/O Events queuing and retries
+
+When an OSGI event of interest is triggered 
+(i.e an OSGI events that matches one of your `OSGI to XDM event mapping configurations`),
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: event (singular), not events (plural)

File: events/using/aem-event-setup.md [hunk 9]
```diff
@@ -411,6 +411,48 @@ To configure using the panel:
...
+
+Note that 
+* Adobe I/O Events `Sling Job Consumer` topic is `com/adobe/eventproxy/events`
+* Adobe I/O Events `Sling Job Queue configuration` is the default `Apache Sling Job Default Queue` with
+  * a Normal job priority
+  * 10 maximum retries
+  * 2 seconds retry delay
+  * 15 maximum parallel jobs
+    
+You can tune this configuration it according to your need: using the `OSGI > configuration` menu of AEM's `system console`
```
Comment (rperelma) [inline events/using/aem-event-setup.md]: You can tune this configuration according to your needs

**Comment by rperelma** [comment]
Thank you for adding the queues and retries section! You may want to add your name to the Authors section at the end, @francoisledroff

File: events/using/aem-event-setup.md (hunk 1 above)
Comment (sarahxxu) [reply events/using/aem-event-setup.md]: How about
"If you do not have administrative permissions, please contact your marketing cloud administrator. After requesting administrative permissions..."

**Comment by sarahxxu** [comment]
it looks good to merge to me! 
@MattLusher could you do a final sweep, merge it to master and make sure it looks as expected on adobe.io? Thanks!

cc @rperelma @francoisledroff @cxs504

**Comment by MattLusher** [comment]
Merged. I'll double-check on the site.

**Pull request merged**